types/passport-google-oauth20/index.d.ts

```diff
@@ -7,10 +7,9 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import * as passport from "passport";
-import * as express from "express";
-import * as oauth2 from "passport-oauth2";
-import { OutgoingHttpHeaders } from "http";
+import passport = require("passport");
```

Comment:
Why did you change the imports? I kinda see no benefit?

Reply:
I saw this import pattern used across other type definition files, so figured I'd make this consistent. Happy to revert this.

Reply:
@ezintz reverted.